doc/devel/coding_guide.rst

```diff
@@ -165,6 +165,48 @@ C/C++ extensions
 
 
 
+PR Review guidelines
+====================
+
+* We run on a 'use your judgement' system (for better or worse) and do
+  not have a strict nose-count rule.  If you have a commit bit, then
```

Comment:
have skbeam / sklearn also looked at doing this with labels?   We do have waffle.io set up which gives some 'auto label' management by moving things between columns.  Could have 3 review related columns 'needs revision', 'needs first review', 'needs final review'.  But that might be getting to fancy.